backend/src/main/java/com/github/devbk84/backend/service/ProductService.java

```diff
@@ -0,0 +1,25 @@
+package com.github.devbk84.backend.service;
+
+import com.github.devbk84.backend.models.Product;
+import com.github.devbk84.backend.repo.ProductRepo;
+import org.springframework.stereotype.Service;
+
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepo productRepo;
+
+    public ProductService(ProductRepo productRepo) {
+        this.productRepo = productRepo;
+    }
+
+    public List<Product> getAllProducts() {
+        return productRepo.findAll();
+    }
+
+    public Product getProductsById(String id) {
```

Comment:
As mentioned above, you get one produtc, not many :)
```suggestion
    public Product getProductById(String id) {
```